docs/guide/tutorial-2-modification-of-the-environment.md

```diff
@@ -154,23 +159,26 @@ graph TD
 %end
 %end
 
-### Add Walls
+> **Hands-on #7**: Create a second ball with the same parameters but using the  [Shape](../reference/shape.md) node (rather than the [Sphere](../reference/sphere.md) node directly) for the DEF-USE mechanism.
 
-For convenience, the `boundingObject` field accepts also the [Shape](../reference/shape.md) node (rather than the [Sphere](../reference/sphere.md) node directly).
-It would be also possible to use the same DEF-USE mechanism at the [Shape](../reference/shape.md) level as shown in [this figure](#def-use-mechanism-applied-on-the-shape-node-of-a-solid).
-For now the greatest benefit is being able to also use this [Shape](../reference/shape.md) directly for graphical purposes.
-Later this mechanism will turn out to be very useful for some sensors.
+
+### Add Walls
 
 In order to verify your progression, implement by yourself four walls to surround the environment.
-The walls have to be defined statically to the environment, and use as much as possible the DEF-USE mechanism at the [Shape](../reference/shape.md) level rather than at the Geometry level.
+The walls have to be defined statically to the environment. To understand the difference between static and dynamic, let's take a defined object (the ball) above the ground. If the [Physics](../reference/physics.md) node is NULL, it will remain frozen in the air during the simulation (static case).
```

Comment:
```suggestion
The walls have to be defined statically to the environment.
To understand the difference between static and dynamic, let's take a defined object (the ball) above the ground.
If the [Physics](../reference/physics.md) node is NULL, it will remain frozen in the air during the simulation (static case).
```